Documentation/Introduction/Index.rst

```diff
@@ -69,3 +69,15 @@ Nginx
 .. code-block:: nginx
 
 	rewrite ^/routing/(.*)$ /index.php?eID=routing&route=$1 last;
+
+HTTP Debug Header
+-----------------
+
+The name of the route is sent as an additional header line in the HTTP response to the client by default:
+
+..
```

Comment:
this needs to be double colons but useless with previous change so just remove this line and the blank one above or below